src/fn/edit-contacts.js

```diff
@@ -135,17 +154,36 @@ function processCsv(docType, cols, row, uuidIndex, toIncludeIndex, documentDocs)
   return doc;
 }
 
-const processUsers =  async (csv, ids, db, args) => {
-  const documentDocs = await fetchDocumentList(db, ids);
-  return  processDocs('user', csv, documentDocs, args);
+const processDocuments =  async (docType, csv, ids, db, args) => {
+  const documentDocs = await fetchDocumentList(db, ids, args);
+  return  processDocs(docType, csv, documentDocs, args);
 };
 
-const processDocuments =  async (csv, ids, db, args) => {
-  const documentDocs = await fetchDocumentList(db, ids);
-  return  processDocs('contact', csv, documentDocs, args);
+const fetchDocumentList = async (db, ids, args) => {
```

Comment:
I think we should pull this out into a separate helper library. 
More actions could benefit from this, including `move-contacts`.
One improvement we could add (maybe later, once the function is used in multiple places) is to add a _rev check where, even if we have a copy of the doc on disk, the server could have an updated version and `upload_docs` would later fail because of a conflict.